plugins/Marketplace/Plugins.php

```diff
@@ -296,7 +296,15 @@ private function enrichPluginInformation($plugin)
             }
         }
 
+        $haDownloadLink = false;
+        if (!empty($plugin['versions'])) {
+            $latestVersion = $plugin['versions'][count($plugin['versions']) - 1];
```

Comment:
Isn't using `array_pop` a bit easier?
```suggestion
            $latestVersion = array_pop($plugin['versions']);
```

Reply:
@sgiehl I [reverted](https://github.com/matomo-org/matomo/pull/22567/commits/7971d96fe337ece5d84ce2d9b68e3de754974387) from `array_pop` as it was leading to description being empty.

Reply:
Ah sure. The versions array is used after that again. And `array_pop` removes the element.
May be a `end()` could do the trick as well.

Reply:
end seems to work :+1: